recipes/gdma/meta.yaml

```
@@ -0,0 +1,137 @@
+{% set name = "gdma" %}
+{% set version = "2.3.3" %}
+{% set commit = "c2e0b548" %}
+{% set sha256 = "87c090b7415cd2126608019cad7a3e008c25171c7c8189adad399b6a86f8d383" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  #url: https://github.com/psi4/gdma/archive/v2.3.3.tar.gz  # equivalent to upstream plus patches
+  url: https://gitlab.com/anthonyjs/{{ name }}/-/archive/{{ commit }}.tar.bz2
+  patches:
+    - 0001-add-CMake-library-pymod-IO-git-avoidance.patch
+  #patches:  # also equivalent to upstream plus single big patch, but c-b predicts patch levels wrong
+  #  - 0001-fix-use-input-bug-build-outside-git-generalize-pytho.patch
+  #  - 0002-add-IO-management-form-library-as-well-as-executable.patch
+  #  - 0003-add-CMake-buildsystem-for-L-M-W-platforms.patch
+  #  - 0004-add-exports-to-Python-module.patch
+  #  - 0005-avoid-git-for-c-f.patch
+  sha256: {{ sha256 }}
```

You only need to set jinja variables for strings used in multiple places or that are modified in some way.

```suggestion
{% set version = "2.3.3" %}
{% set commit = "c2e0b548" %}

package:
  name: gdma-split
  version: {{ version }}

source:
  #url: https://github.com/psi4/gdma/archive/v2.3.3.tar.gz  # equivalent to upstream plus patches
  url: https://gitlab.com/anthonyjs/gdma/-/archive/{{ commit }}.tar.bz2
  patches:
    - 0001-add-CMake-library-pymod-IO-git-avoidance.patch
  #patches:  # also equivalent to upstream plus single big patch, but c-b predicts patch levels wrong
  #  - 0001-fix-use-input-bug-build-outside-git-generalize-pytho.patch
  #  - 0002-add-IO-management-form-library-as-well-as-executable.patch
  #  - 0003-add-CMake-buildsystem-for-L-M-W-platforms.patch
  #  - 0004-add-exports-to-Python-module.patch
  #  - 0005-avoid-git-for-c-f.patch
  sha256: 87c090b7415cd2126608019cad7a3e008c25171c7c8189adad399b6a86f8d383
```

---

Good point on the name. The `sha256` I sometimes keep near the version and commit so that I remember to change them together. I'll tidy them up at the feedstock. Thanks for the merge.